client/src/containers/CampaignAdmin/CampaignAdminList.js

```diff
@@ -114,14 +137,25 @@ class CampaignAdminList extends SilverStripeComponent {
           itemClassNames.push('active');
         }
 
+        let isLinked = !!selectedItemsLinkedTo.find(
+            linkToObj => linkToObj.ID === parseInt(item.ObjectID, 10));
+
+        isLinked = isLinked || selectedItemsLinkedFrom.find(
```

Comment:
Why do you only show selectedItemsLinkedFrom if selectedItemsLinkedTo is empty?

As an analogy, this would be only showing child pages in the site tree for pages that have no parent.